sound/soc/sof/intel/hda.c

```diff
@@ -1175,6 +1175,9 @@ static void hda_generic_machine_select(struct snd_sof_dev *sdev,
 #endif
 
 #if IS_ENABLED(CONFIG_SND_SOC_SOF_INTEL_SOUNDWIRE)
+
+#define SDW_ADR_EX_UNIQUE_ID(_adr) (_adr & ~SDW_UNIQUE_ID_MASK)
```

Comment:
can we rename this to SDW_UNIQUE_ID_STRIPPED_ADR to be more intuitive?

Reply:
> can we rename this to SDW_UNIQUE_ID_STRIPPED_ADR to be more intuitive?

Done